test/services/offer.service.test.js

```diff
@@ -484,16 +484,16 @@ describe('Offer Service', () => {
 
     context('when all is valid', () => {
       it('returns a valid cancelled offer', async () => {
-        const rejectedOffer = await cancelOffer(offerId);
-        expect(rejectedOffer.status).to.eql('Cancelled');
+        const rejectedOffer = await cancelOffer({ offerId, vendorId: userId });
+        expect(rejectedOffer.status).to.eql(OFFER_STATUS.CANCELLED);
```

Comment:
This looks good but a Senior Software engineer had once advised me that it is not best practices for test to use defined constants. His reason is that the test will still pass if the constant is mistakenly updated e.g if someone updates constant.js file and added an unknown character into this, something like 'Cancelled2', the test will still pass.

I think it is a valid reason, even though I knowingly ignore the past constants you used in previous tests to minimize the number of comments on your PR. Henceforth, lets try to minimize or use the real value instead of constants. 
```suggestion
        expect(rejectedOffer.status).to.eql('Cancelled');
```